crates/web5/src/dids/bearer_did.rs

```diff
@@ -9,7 +9,7 @@ use super::{
 use crate::{
     crypto::{
         dsa::Signer,
-        key_managers::{in_memory_key_manager::InMemoryKeyManager, key_manager::KeyManager},
+        key_managers::{in_memory_key_manager::InMemoryKeyManager, KeyManager},
```

Comment:
we can address export hierarchy/API in a separate PR, but mentioning just to put the idea out there: `key_managers::in_memory_key_manager::InMemoryKeyManager` is a lot of times to say "key manager" before I get to the thing I need. Luckily rust makes it easy to export things under the exact namespace we want while still keeping or underlying file hierarchy the same.

Some musings about namespaces in [this talk about python](https://www.youtube.com/watch?v=o9pEzgHorH0&t=567s) may also apply in rust. Namely, namespaces are best for preventing collisions, and can become cumbersome when used to create taxonomies.

Reply:
I 100% agree and it would be awesome if you want to spearhead @diehuxx 

IMO we should go as far as to define namespace in the APID such that all projects have the same namespacing